### PR TITLE
fix softmax_with_cross_entropy en docs

### PR DESCRIPTION
**Pull request by yghstill**
### PR types

Others
### PR changes

Docs
### Describe

fix softmax_with_cross_entropy en docs.

### CODE REVIEW

**Review by momozi1996 — DISMISSED** [review]
LGTM

File: python/paddle/nn/functional/loss.py [hunk 1]
```diff
@@ -2030,6 +2030,99 @@ def softmax_with_cross_entropy(logits,
...
+    one class with a probability of 1.0. Each sample in the batch will have a 
+    single label.
+
+    The equation is as follows:
+
+    1) Hard label (one-hot label, so every sample has exactly one class)
+
+    .. math::
+
+        loss_j =  -\\text{logits}_{label_j} +
```
Comment (momozi1996) [inline python/paddle/nn/functional/loss.py]: 公式的语法不太对，显示有问题

File: python/paddle/nn/functional/loss.py [hunk 2]
```diff
@@ -2030,6 +2030,99 @@ def softmax_with_cross_entropy(logits,
...
+
+    .. math::
+
+        loss_j =  -\\sum_{i=0}^{K}\\text{label}_i
+        \\left(\\text{logits}_i - \\log\\left(\\sum_{i=0}^{K}
+        \\exp(\\text{logits}_i)\\right)\\right), j = 1,...,K
+
+    3) If :attr:`numeric_stable_mode` is :attr:`True`, softmax is calculated first by:
+
+    .. math::
```
Comment (momozi1996) [inline python/paddle/nn/functional/loss.py:2053]: 公式语法格式有问题

File: python/paddle/nn/functional/loss.py [hunk 3]
```diff
@@ -2030,6 +2030,99 @@ def softmax_with_cross_entropy(logits,
...
+    The equation is as follows:
+
+    1) Hard label (one-hot label, so every sample has exactly one class)
+
+    .. math::
+
+        loss_j =  -\\text{logits}_{label_j} +
+        \\log\\left(\\sum_{i=0}^{K}\\exp(\\text{logits}_i)\\right), j = 1,..., K
+
+    2) Soft label (each sample can have a distribution over all classes)
```
Comment (momozi1996) [inline python/paddle/nn/functional/loss.py:2046]: 公式语法格式有问题

**Review by momozi1996 — COMMENTED** [review]
公式的语法，需要修改一下

File: python/paddle/nn/functional/loss.py (hunk 1 above)
Comment (yghstill) [reply python/paddle/nn/functional/loss.py]: done

File: python/paddle/nn/functional/loss.py (hunk 3 above)
Comment (yghstill) [reply python/paddle/nn/functional/loss.py:2046]: done

**Review by momozi1996 — APPROVED** [review]
image
预览链接中，英文文档的公式暂时没有问题了。代码块也可以正常显示了。

**Review by ceci3 — APPROVED** [review]
